BSB_lan_defs.h

```diff
@@ -8250,8 +8301,10 @@ PROGMEM_LATE const cmd_t cmdtbl2[]={
 {0x073D040E,  CAT_IOTEST,           VT_PERCENT,       7724,  STR7724,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Ausgangstest UX3 //FUJITSU
 {0x073D1238,  CAT_IOTEST,           VT_VOLTAGE,       7725,  STR7725,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // [V ] - Ein-/Ausgangstest - Spannungssignal U4 (Ux3) //FUJITSU
 {0x053D0470,  CAT_IOTEST,           VT_TEMP,          7730,  STR7730,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // [°C ] - Ein-/Ausgangstest - Aussentemperatur B9
-{0x053D08C5,  CAT_IOTEST,           VT_TEMP,          7732,  STR7732,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // [°C ] - Ein-/Ausgangstest - vorlauftemperatur B1
-{0x053D08C6,  CAT_IOTEST,           VT_TEMP,          7734,  STR7734,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // [°C ] - Ein-/Ausgangstest - Vorlauftemperatur B12
+// FL_NO_CMD - clash with 7814, 7973
+{0x053D08C5,  CAT_IOTEST,           VT_TEMP,          7732,  STR7732,  0,                    NULL,         FL_NO_CMD,     DEV_ALL}, // [°C ] - Ein-/Ausgangstest - vorlauftemperatur B1
```

Comment:
The clashing commands are:
7732 (DEV_ALL) 	7814	(7973  DEV_170_ALL) (hides my command)
7734 (DEV_ALL) 	7807 	(hides my command)
7820 (DEV_ALL) 	7804 
7821 (DEV_ALL) 	7805
7822 (DEV_ALL) 	7806
7884 (DEV_ALL) 	7946	(hides my command)
In first column you have existing commands with the same command ID as commands in the second column. They are defined DEV_ALL. If now I add for example 7814 (DEV_108_ALL), then it will be hidden by 7732. 
If you remember, in my previous pull request I solved problem with clash, by change of DEV_ALL to "DEV_ALL - DEV_108_ALL" (new lines for all families except DEV_108_ALL). As you wanted I switched now to FL_NO_CMD without thinking too much :)

If there is knowledge for which families commands from the first column are valid, then they should be assigend to the proper family, if not then switch from DEV_ALL to "DEV_ALL - DEV_108_ALL" is probably the only solution - or?

You could check in "Albatros Wärmepumpenregler Benutzerhandbuch RVS61.843" (Google search "CE1U2355de_052") that parameters from second column are valid for my controller.

BX11 in my case is not connected. "Vorlauftemperatur" - in my case "WP Vorlauffühler B21" is  connected to BX10.

_7973  DEV_170_ALL was already existing and I could assume that it was hidden by 7732 too._

Reply:
It is no problem when there are different parameter numbers with the same command id. They don't "hide" any other command. Try it out: Leave 7734 as it is with DEV_ALL and add 7807 with DEV_108_ALL or even DEV_ALL. If you then call http://<ip>/7807 you should see your parameter displayed correctly. 
As I said: "If a specific command id clashes with an existing parameter AND if the meaning of these parameters is not just semantically different, ONLY THEN use FL_NO_CMD for your device family only." This is only in very few cases necessary, for example in the Error section. There we have several heaters which use the same command ID, but on one heater it's history entry 4 and at the other it's history entry 1 (or something like that). Now, that is of course misleading and therefore we use FL_NO_CMD there to "hide" the "wrong" command id/parameter combination for that machine.

For memory reasons, we can't provide a "clean" list of parameters for every heater. If it should happen that the web interface lists parameter 7734 with a valid value and you don't have parameter 7734 on your heater, that is not a problem (to us ;) ), as long as you get 7807 with the correct parameter.